### PR TITLE
Provide spec file for rpmbuild

### PR DESCRIPTION
**Pull request by jschmidb**
Provide new spec file for rpmbuild and change install location
of libzpc.pc to libdir.

Signed-off-by: Joerg Schmidbauer <jschmidb@de.ibm.com>

### CODE REVIEW

**Comment by ifranzki** [comment]
is the 2nd commit "[Merge branch 'main' into spec_file](https://github.com/opencryptoki/libzpc/pull/6/commits/d0bdc1cac4a5ef068824ef615effb10ce7bfc054)" intended ?

File: libzpc.spec [hunk 1]
```diff
@@ -47,7 +47,7 @@ The %{name}-static package contains the static library of %{name}.
 
 
 %prep
-%setup -q -n %{name}
+%autosetup -q -n %{name}-%{version}
```
Comment (sharkcz) [inline libzpc.spec:50]: This should be just %autosetup, the default behaviour is `-q -n ...`.

File: libzpc.spec [hunk 2]
```diff
@@ -74,13 +74,17 @@ The %{name}-static package contains the static library of %{name}.
 %files
 %doc README.md CHANGES.md
 %license LICENSE
-%{_libdir}/%{name}.so
+%{_libdir}/%{name}.so.%{soversion}*
```
Comment (sharkcz) [inline libzpc.spec:77]: OK

File: libzpc.spec [hunk 3]
```diff
@@ -74,13 +74,17 @@ The %{name}-static package contains the static library of %{name}.
...
 
 
 %files devel
 %{_includedir}/zpc/
 %{_libdir}/pkgconfig/%{name}.pc
-%{_libdir}/%{name}.so.%{soversion}*
+%{_libdir}/%{name}.so
```
Comment (sharkcz) [inline libzpc.spec:83]: OK

File: libzpc.spec [hunk 4]
```diff
@@ -74,13 +74,17 @@ The %{name}-static package contains the static library of %{name}.
...
+
+
+%files static
```
Comment (sharkcz) [inline libzpc.spec:86]: OK

File: libzpc.spec [hunk 5]
```diff
@@ -3,7 +3,7 @@ Version:	1.0.0
 Release:	1%{?dist}
 Summary:	Open Source library for the IBM Z Protected-key crypto feature
 
-License:	MIT License
+License:	MIT
```
Comment (sharkcz) [inline libzpc.spec:6]: OK

**Review by sharkcz — CHANGES_REQUESTED** [review]

File: libzpc.spec (hunk 1 above)
Comment (jschmidb) [reply libzpc.spec]: Done.

**Pull request merged**